doc/liblouis.texi

```diff
@@ -1169,6 +1169,19 @@ This is an opcode prefix which modifies the operation of the opcode
 following it on the same line. nofor specifies that forward translation 
 is not to use the information on this line.
 
+@opcode{nocross, opcode characters ...}
+@code{nocross} is an opcode prefix which modifies the operation of the
```

Comment:
I'd rephrase this as something along the lines of:
``` texi
which modifies the behaviour of the opcode following it. A typical example would be the 
@opcoderef{always} or the translation opcode with @code{characters} as first operand. 
All valid opcodes that can be preceeded with @code{nocross} are: ...

@code{nocross} specifies that the operation ...
```

Reply:
OK, except that I didn't want to explicitly list all the valid opcodes because:
- it's a long list. Better would be to make a stricter categorization of opcodes in the documentation so that we can simply reference a section.
- I already said "any translation opcode listed below with characters as the first operand" which is more or less what you can use, and typically what you would use. There might however be cases that don't work, I haven't tested them all. Also there might be some other opcodes (that don't have "characters" as the first operand) that might work.